model_analyzer/config/input/config_command_profile.py

```diff
@@ -483,6 +483,16 @@ def _add_run_search_configs(self):
                         parser_args={'action': 'store_true'},
                         default_value=DEFAULT_RUN_CONFIG_SEARCH_DISABLE,
                         description="Disable run config search."))
+        self._add_config(
+            ConfigField(
+                'run_config_profile_models_concurrently_enable',
+                flags=['--run-config-profile-models-concurrently-enable'],
+                field_type=ConfigPrimitive(bool),
+                parser_args={'action': 'store_true'},
+                default_value=
+                DEFAULT_RUN_CONFIG_PROFILE_MODELS_CONCURRENTLY_ENABLE,
+                description=
+                "Enable the profiling of multiple models concurrently."))
```

Comment:
We only support all or nothing, right? I think the description should say "profiling of all supplied models concurrently"